src/platforms/macos/daemon/macosroutemonitor.cpp

```diff
@@ -95,6 +95,10 @@ void MacosRouteMonitor::handleRtmDelete(const struct rt_msghdr* rtm,
       !(rtm->rtm_addrs & RTA_NETMASK) || (addrlist.count() < 3)) {
     return;
   }
+  // Ignore interface-scoped routes.
```

Comment:
Could you expand this comment a tad more to describe why we are ignoring these flags?